asdf_astropy/converters/utils.py

```diff
@@ -18,3 +18,26 @@ def import_type(type_name):
     """
     module_name, class_name = type_name.rsplit(".", 1)
     return getattr(importlib.import_module(module_name), class_name)
+
+
+# FIXME: ASTROPY_LT_7_1: Remove once we depend on astropy >= 7.1.
+class MaskedClassesRequireAstropy71Error(NotImplementedError):
```

Comment:
I don't think this needs a custom exception or the astropy version.

Reply:
This is now just an ordinary `NotImplementedError`. Please take a look.